detekt-test/src/main/kotlin/io/gitlab/arturbosch/detekt/test/RuleExtensions.kt

```diff
@@ -72,7 +72,7 @@ fun Rule.lint(ktFile: KtFile, compilerResources: CompilerResources = FakeCompile
 
 private fun List<Finding>.filterSuppressed(rule: Rule): List<Finding> =
     filterNot {
-        it.entity.ktElement.isSuppressedBy(rule.ruleName.value, rule.aliases, RuleSet.Id("NoARuleSetId"))
+        it.entity.ktElement.isSuppressedBy(Rule.Id(rule.ruleName.value), rule.aliases, RuleSet.Id("NoARuleSetId"))
```

Comment:
does this have a typo in it?

```suggestion
        it.entity.ktElement.isSuppressedBy(Rule.Id(rule.ruleName.value), rule.aliases, RuleSet.Id("NotARuleSetId"))
```
```suggestion
        it.entity.ktElement.isSuppressedBy(Rule.Id(rule.ruleName.value), rule.aliases, RuleSet.Id("NoRuleSetId"))
```